src/answers.sql

```diff
@@ -1,15 +1,87 @@
 -- Your answers here:
 -- 1
-
--- 2
-
+SELECT c.name,
+    COUNT(c.name)
+FROM countries c
+    INNER JOIN states s ON c.id = s.country_id
+GROUP BY c.name;
+--2
+SELECT COUNT(*) employees_without_bosses
+FROM employees
+WHERE supervisor_id IS NULL;
 -- 3
-
+SELECT c.name,
+    o.address,
+    COUNT(e.id) as count
+FROM employees e
+    INNER JOIN offices o ON e.office_id = o.id
+    INNER JOIN countries c ON c.id = o.country_id
+GROUP BY c.name,
+    o.address
+ORDER BY count DESC,
+    c.name
+LIMIT 5;
 -- 4
-
+SELECT supervisor_id,
+    COUNT(supervisor_id) as count
+FROM employees
+GROUP BY supervisor_id
+ORDER BY count DESC
+LIMIT 3;
 -- 5
-
+SELECT COUNT(*) list_of_office
```

Comment:
Naming could be improved. I propose `offices_in_colorado`

Reply:
Changed!